app/src/main/java/de/tum/in/tumcampusapp/component/tumui/lectures/model/Lecture.kt

```diff
@@ -10,21 +12,21 @@ import de.tum.`in`.tumcampusapp.component.other.generic.adapter.SimpleStickyList
  */
 @Xml(name = "row")
 data class Lecture(
-    @PropertyElement(name = "dauer_info") val duration: String = "",
-    @PropertyElement(name = "org_kennung_betreut") val chairTumId: String? = null,
-    @PropertyElement(name = "org_name_betreut") val chairName: String? = null,
+    @PropertyElement(name = "dauer_info", converter = EscapedStringConverter::class) val duration: String = "",
```

Comment:
Maybe we should just make these either all null or all empty string?

Reply:
From a brief overview of the consumers of the lecture api, this would not be my approach, as it is conceivable for an endpoint to return either null, indicating that an error message should be displayed, or an empty string which should be interpreted as just that: nothing to display.

On the other hand it might be wise to clean up our consumers to handle both cases the same way, but this would require a more in depth knowledge of the api and intended output which I do not have.

Reply:
Ack, got it.